File: QuantConnect.AlphaStream/Models/Point.cs
using Newtonsoft.Json;

namespace QuantConnect.AlphaStream.Models
{
    /// <summary>
    /// A time-value pair
    /// </summary>
    public class Point
    {
        /// <summary>
        /// Time value of a time-value pair.
        /// </summary>
        [JsonProperty("time")]
        public long Time { get; set; }

        /// <summary>
        /// Value of the point.
        /// </summary>
        [JsonProperty("value")]
        public double Value { get; set; }
    }
}

File: QuantConnect.AlphaStream.Python/Models/Point.py
from datetime import datetime


class Point:
    """Time-value pair"""

    def __init__(self, json):
        if not isinstance(json, dict):
            self.Time = None
            self.Value = None
            return

        self.Time = datetime.utcfromtimestamp(json['time']) if 'time' in json else None
        self.Value = json.get('value', None)